swift/downgrades/264374a5a3d4445024add4c63e657b053f6165e8/upgrade.properties

```diff
@@ -0,0 +1,5 @@
+description: revert removal of IterableDeclContext
+compatibility: partial
```

Comment:
Is it fair to assume that any `Decl` with members is an `IterableContextDecl` after the downgrade?  If so, do we actually lose anything in the downgrade?

Reply:
Yes, the downgrade script specifies that only `@nominal_type_decl or @extension_decl` (i.e. the old `@iterable_context_decl`) will revert back to being an IterableContextDecl with members. What we could lose are any future Decl subtypes that override the `getImmediateMember()` predicate, for instance.

I've been having trouble testing the db upgrade/downgrade script though; against the same baseline (upstream/main more or less), I can only successfully run the upgrade script, not the downgrade script. I've been following the steps from [here](https://github.com/github/codeql/blob/main/docs/prepare-db-upgrade.md).

Reply:
Yes, I have a similar situation in https://github.com/github/codeql/pull/12001 - I can test the upgrade script but I don't know how to test the downgrade script.  I'll ask around if anyone knows how to do this...

Reply:
It should run fine now 🤞🏻 Managed to test the upgrade/downgrade scripts.